components/VcItemContent.tsx

```diff
@@ -96,38 +110,35 @@ export const VcItemContent: React.FC<VcItemContentProps> = props => {
     <ImageBackground
       source={!props.verifiableCredential ? null : Theme.CloseCard}
       resizeMode="stretch"
-      borderRadius={4}
       style={
         !props.verifiableCredential
           ? Theme.Styles.vertloadingContainer
           : Theme.Styles.backgroundImageContainer
       }>
       <Column>
         <Row align="space-between">
-          <Row>
+          <Row margin="5 0 0 5">
             <ImageBackground
+              imageStyle={Theme.Styles.faceImage}
               source={
                 !props.verifiableCredential
                   ? Theme.ProfileIcon
                   : {uri: props.context.credential.biometrics.face}
               }
               style={Theme.Styles.closeCardImage}>
               {props.iconName && (
-                <Icon
-                  {...testIDProps('pinIcon')}
-                  name={props.iconName}
-                  type={props.iconType}
-                  color={Theme.Colors.Icon}
-                  style={{marginLeft: -80}}
-                />
+                <Image
+                  source={Theme.PinIcon}
+                  style={Theme.Styles.pinIcon}></Image>
```

Comment:
For images no need to use end-tags.
Just pass required props within **<Image/>**

Reply:
Removed the closing tag